fiddle/extensions/tf_test.py

```diff
@@ -69,13 +69,20 @@ def test_special_value_codegen(self):
   def test_codegen(self):
     config = fdl.Config(foo, dtype=tf.bfloat16)
     code = "\n".join(codegen.codegen_dot_syntax(config).lines())
-    expected = """
+    if __name__ == '__main__':
```

Comment:
The rest of this file uses double quotes instead of single quotes, causing a lint error.

Reply:
Great point. Thanks!